styles/Canonical/021b-latinisms-to-reconsider.yml

```diff
@@ -0,0 +1,19 @@
+# 021 - Latinisms that we strongly advise against
+# Since Vale substitutions can't ignore word boundaries, meaning they can't detect the punctuation in "i.e." and "&c", we merely check for them and then notify the user.
```

Comment:
@evilnick This part was awkward. I'd rather include lines 14-17 in the replacement table. Do you know a workaround for including punctuation that's normally counted as a word boundary?

Reply:
Ah, I could swear I tried that. Thanks!

3e52fdb10ef5759311fcdf5e49371afd4f8ea9a6